casparser/process/cas_detailed.py

```diff
@@ -9,20 +9,18 @@
 from ..exceptions import HeaderParseError, CASParseError
 from .regex import AMC_RE, DETAILED_DATE_RE, FOLIO_RE, SCHEME_RE, REGISTRAR_RE
 from .regex import CLOSE_UNITS_RE, NAV_RE, OPEN_UNITS_RE, VALUATION_RE, DESCRIPTION_TAIL_RE
-from .regex import DIVIDEND_RE, TRANSACTION_RE1, TRANSACTION_RE2, TRANSACTION_RE3
+from .regex import DIVIDEND_RE, TRANSACTION_RE1, TRANSACTION_RE2, TRANSACTION_RE3, TRANSACTION_RE4
 from ..types import FolioType, SchemeType
 from .utils import isin_search
 
 ParsedTransaction = namedtuple(
     "ParsedTransaction", ("date", "description", "amount", "units", "nav", "balance")
 )
 
-
```

Comment:
The repo uses [black](https://github.com/psf/black) for formatting and these changes will break certain pre-build tests. 

Please run black formatter on the files (You can also configure [pre-commit](https://pre-commit.com) to make sure the committed files are auto-formatted according to the guidelines)